pandoc.cabal

```diff
@@ -180,6 +181,7 @@ Extra-Source-Files:
                  test/writer.markdown
                  test/writer.plain
                  test/writer.mediawiki
+                 test/writer.muse
```

Comment:
I don't have it actually. I added it when I began development, but removed it then as I thought it was deprecated as it was referred from `test/Tests/Old.hs`. Instead I made a comprehensive unit test suite.

Should I add it or are we moving towards unit tests?